addOns/fuzz/src/main/java/org/zaproxy/zap/extension/fuzz/httpfuzzer/HttpFuzzerHandler.java

```diff
@@ -147,7 +146,9 @@ private HttpFuzzer createFuzzer(
 
         MessageLocationReplacer<HttpMessage> replacer =
                 MessageLocationReplacers.getInstance()
-                        .getMLR(HttpMessage.class, TextHttpMessageLocation.class);
+                        .getMLR(
+                                HttpMessage.class,
+                                fuzzLocations.get(0).getMessageLocation().getClass());
```

Comment:
Hi @thc202 ,

I am not sure how but the FuzzLocations is returning the instance of DefaultHttpMessageLocation for existing fuzzer instead of TextHttpMessageLocation which is the registered one and hence they are failing. Need to debug more.

thanks,
Karan